dev/TabView/APITests/TabViewTests.cs

```diff
@@ -167,6 +168,113 @@ static ISelectionItemProvider GetProviderFromTVI(TabViewItem item)
+                    return brush is SolidColorBrush scBrush && scBrush.Color == color;
+                }
+            });
+        }
+
+        [TestMethod]
+        public void TabViewItemForegroundTest()
+        {
+            TabView tabView = null;
+            TabViewItem tabViewItem1 = null;
+            TabViewItem tabViewItem2 = null;
+            RunOnUIThread.Execute(() =>
+            {
+                tabView = new TabView();
+
+                tabViewItem1 = CreateTabViewItem("Tab1", Symbol.Home);
+                tabViewItem2 = CreateTabViewItem("Tab2", Symbol.Document);
+
+                tabView.TabItems.Add(tabViewItem1);
+                tabView.TabItems.Add(tabViewItem2);
+
+                Content = tabView;
+            });
+
+            IdleSynchronizer.Wait();
+
+            RunOnUIThread.Execute(() =>
+            {
+                // Verify that the TabViewItem we use for foreground API testing here is unselected
+                Verify.IsFalse(tabViewItem2.IsSelected, "TabViewItem should have been unselected");
+
+                bool testCondition = !IsBrushUsingSolidColor(tabViewItem2.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "Default TabViewItem foreground color should have not been [blue]");
+
+                var iconControl = VisualTreeUtils.FindVisualChildByName(tabViewItem2, "IconControl") as ContentControl;
+                var headerPresenter = VisualTreeUtils.FindVisualChildByName(tabViewItem2, "ContentPresenter") as ContentPresenter;
+
+                testCondition = !IsBrushUsingSolidColor(iconControl.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [IconControl] foreground color should have not been [blue]");
+
+                testCondition = !IsBrushUsingSolidColor(headerPresenter.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [ContentPresenter] foreground color should have not been [blue]");
+
+                Log.Comment("Set the foreground color of the TabViewItem to [blue]");
+                tabViewItem2.Foreground = new SolidColorBrush(Colors.Blue);
+
+                testCondition = IsBrushUsingSolidColor(tabViewItem2.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's foreground color should have been [blue]");
+
+                testCondition = IsBrushUsingSolidColor(iconControl.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [IconControl] foreground color should have been [blue]");
+
+                testCondition = IsBrushUsingSolidColor(headerPresenter.Foreground, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [ContentPresenter] foreground color should have been [blue]");
+
+                bool IsBrushUsingSolidColor(Brush brush, Color color)
+                {
+                    return brush is SolidColorBrush scBrush && scBrush.Color == color;
+                }
```

Comment:
you could pull this out right?

Reply:
Briefly thought about that because I am using code like this on quite a few occasations already...and I suspect this wasn't the last time. 

I could create a new, say, `BrushExtensions` class in [TestHelpers.cs](https://github.com/microsoft/microsoft-ui-xaml/blob/master/test/MUXControlsTestApp/Utilities/TestHelpers.cs) as it already contains a class like `ComboBoxExtensions`.

Reply:
Sounds cool, but not necassary.

Reply:
So just keep it as it is right now then?

Reply:
sure